src/app/main/component/shared/components/edit-photo-pop-up/edit-photo-pop-up.component.ts

```diff
@@ -38,6 +38,11 @@ export class EditPhotoPopUpComponent implements OnInit {
   }
 
   public onSelectPhoto(event): void {
+    if (event.target.files[0].size >= 10000000) {
+      this.matDialogRef.close();
+      this.snackBar.openSnackBar('update-cafe.notifications.photoUpload');
+      return;
+    }
```

Comment:
maybe we should return the error in case if a file is bigger?

Reply:
done